src/test/java/com/training/baseTest/FlipkartBaseTest.java

```diff
@@ -0,0 +1,34 @@
+package com.training.baseTest;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.util.concurrent.TimeUnit;
+
+public class FlipkartBaseTest {
+    public WebDriver driver;
+    @BeforeEach
+    public void setup(){
+        System.setProperty("webdriver.chrome.driver", "package/resources/chromedriver.exe");
+        ChromeOptions options = new ChromeOptions().addArguments("incognito");
+        DesiredCapabilities capabilities = DesiredCapabilities.chrome();
+        capabilities.setCapability(ChromeOptions.CAPABILITY, options);
+        driver = new ChromeDriver(capabilities);
+
+        /**
+         * Launch Website and maximise
+         */
+        driver.navigate().to("https://www.flipkart.com/");
```

Comment:
This can rather be defined in your test class as this would be test specific. For flipkart the url would be this, but for any other sites, url would be different